java/fury-core/src/test/java/org/apache/fury/FuryCopyTest.java

```diff
@@ -137,8 +137,10 @@ public void threadLocalCopyTest() {
     BeanA beanA = BeanA.createBeanA(2);
     ExecutorService executor = Executors.newSingleThreadExecutor();
     AtomicReference<Throwable> ex = new AtomicReference<>();
-    ThreadLocalFury threadLocalFury =
-        builder().withCodegen(false).withRefCopy(true).buildThreadLocalFury();
+    ThreadLocalFury threadLocalFury = builder().withCodegen(false).withRefCopy(true).buildThreadLocalFury();
+    threadLocalFury.getClassResolver().setClassChecker((classResolver, className1) -> true);
```

Comment:
This won't set set class checker for other fury instance. We could set Class checker when building threadsafefury by creating threadlocalfury directly

Reply:
> This won't set set class checker for other fury instance. We could set Class checker when building threadsafefury by creating threadlocalfury directly

I believe that in some cases, adding check listeners and serializers involves dynamically adding or modifying them at runtime, which cannot be predicted with absolute certainty

Reply:
If so, maybe we need to add `setClassChecker` to BaseFury/ThreadSafeFur directly.

Reply:
> If so, maybe we need to add `setClassChecker` to BaseFury/ThreadSafeFur directly.

That's a good idea. I will make modifications based on this suggestion

Reply:
done